pkg/reconciler/taskrun/resources/taskref_test.go

```diff
@@ -182,7 +183,7 @@ func TestLocalTaskRef(t *testing.T) {
 					Namespace: "default",
 				},
 			},
-			wantErr: false,
+			wantErr: nil,
```

Comment:
This looks like unnecessary

Reply:
I think for the indication that the wanted error is nil, it might make sense to be kept here for easier interpretation of the test case. otherwise one would need to check if there's a wantErr looking at the current test cases ie https://github.com/tektoncd/pipeline/blob/5e93dd313fa7aa82029a5f33fc20b4b4de9df9ba/pkg/status/status_test.go#L341. WDYT?